assets/css/style.css

```diff
@@ -172,4 +173,227 @@ body {
 #nav-checkbox:checked ~ .navigation {
   display: flex;
   flex-direction: column;
+  animation: slide-right 1s ease;
+  left: 0;
+}
+
+#nav-checkbox:not(:checked) ~ .navigation {
+  animation: slide-left 1s ease;
+  left: -100%;
+  display: none;
+}
+
+@keyframes slide-right {
+  0% {
+    left: -100%;
+  }
+
+  100% {
+    left: 0;
+  }
+}
+
+@keyframes slide-left {
+  0% {
+    left: 0;
+  }
+
+  100% {
+    left: -100%;
+  }
+}
+
+.achieved-work {
+  background-color: #fff;
+  padding: 0 24px;
+}
+
+.section-title {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  margin-top: 50px;
+  margin-bottom: 24px;
+}
+
+.section-title-separators {
+  border: 3px solid #36b37e;
+  width: 48px;
+  margin-bottom: 48px;
+}
+
+.card {
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  overflow: hidden;
+  margin: 20px auto;
+}
+
+.project-description {
+  padding: 16px;
+  text-align: center;
+  border-top: 1px solid #d0d9d4;
+  border-top-left-radius: 8px;
+  border-top-right-radius: 8px;
+}
+
+.project-description h3 {
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  margin: 16px auto;
+}
+
+.achievementlist {
+  list-style-type: none;
+  display: grid;
+  padding: 0;
+}
```

Comment:
- [ ] Great job using `display: grid` as required, but right now, your project is displaying one work per row. In order to match the original design better, you should consider styling them to be displayed 3 per row, with at least 2 rows.

Figma design:
![image](https://user-images.githubusercontent.com/66967274/143451112-a886c1fb-d87a-40f2-887c-12c550376034.png)


Your design:
![image](https://user-images.githubusercontent.com/66967274/143451019-9405197b-2008-46e0-9ee1-aeee9f1fa379.png)

Reply:
But we are only working on the mobile version of the web